test-java-lib/src/main/java/com/uber/lib/unannotated/UnannotatedWithModels.java

```diff
@@ -2,6 +2,10 @@
 
 public class UnannotatedWithModels {
 
+  public Object nullableFieldUnannotated1;
+
+  public Object nullableFieldUnannotated2;
```

Comment:
What is the purpose of having two fields here?  Are we testing something different with each field?

Reply:
Added in 819f75e48fa87ef50eae02ec008fc19fbdfbcadf

Reply:
Doesn't hurt anything I guess, we can leave it :-)